changelog.txt

```diff
@@ -1,7 +1,8 @@
 0.83
  * BREAKING CHANGE: Database file format update. The Realm file created by this version cannot be used by previous versions of Realm.
  * BREAKING CHANGE: Removed deprecated methods and constructors from the Realm class.
-  * Fixed an issue where opening the same Realm file on two Looper threads could potentially lead to an IllegalStateException being thrown.
+ * BREAKING CHANGE: Introduced boxed types Boolean, Byte, Short, Integer, Long, Float and Double. Added null support. Introduced annotation @Required to indicate a field is not nullable. String, Date and byte[] became nullable by default which means a RealmMigrationNeededException will be thrown while trying to open a previous version Realm db file.
```

Comment:
Well... The others are precondition of the String/Date. But I need to put the BREAKING CHANGE in front. It is a dead lock condition. I would let user to notice it well....